### ISSUE TITLE
Download a given release?

### ISSUE THREAD

**Issue by cponder** [JohnWStockwellJr/SeisUnix#70]
Is there a path rule that I can use to download a particular release version of SeisUnix, something similar to

    https://....../SeisUnix/v44R14/distro.tar,gz

It looks like you don't post releases on this site.
Also you mention 44R14 as being the snapshot, while this site says it has 44R18:

    https://nextcloud.seismic-unix.org/index.php/s/WgXpRba4DPPaiNK

(Note also that this path uses some hash garbage instead of a version number, so I can't write rules for downloading specified versions...)

**Comment by JohnWStockwellJr** [comment]
44R18 is the current incremental release. Past incremental releases are not available at this time. Was there something specific you wanted from 44R14?

John Stockwell
Visit: https://wiki.Seismic-Unix.org
The new home of Seismic Un*x

**Comment by cponder** [comment]
No, but I'd like to be able to download using the same rule going forward.
This page does post some incremental releases (though it may just be someone's mirror-for-the-moment)

     https://nextcloud.seismic-unix.org/index.php/s/LZpzc8jMzbWG9BZ

and for now, it looks like this rule will work

    wget "https://nextcloud.seismic-unix.org/s/LZpzc8jMzbWG9BZ/download?path=/&files=cwp_su_all_$(VERSION).tgz" -O cwp_su_all_$(VERSION).tgz

though I don't know if I can expect this "LZpzc8jMzbWG9BZ" to be stable.

**Comment by JohnWStockwellJr** [comment]
The download site may be found at this address

https://wiki.Seismic-Unix.org

You go there and click on the link.

John Stockwell
Visit: https://wiki.Seismic-Unix.org
The new home of Seismic Un*x

**Comment by cponder** [comment]
How do i do this with "wget" though? I'm trying to set up automatic download/build.

**Comment by cponder** [comment]
This link here doesn't encode any version information:

https://nextcloud.seismic-unix.org/index.php/s/WgXpRba4DPPaiNK

**Comment by JohnWStockwellJr** [comment]
Does this address work with wget?
https://nextcloud.seismic-unix.org/index.php/s/WgXpRba4DPPaiNK


John Stockwell
Visit: https://wiki.Seismic-Unix.org
The new home of Seismic Un*x

**Comment by cponder** [comment]
Actually no, it gave me an HTML file.
But this one works:

     wget "https://nextcloud.seismic-unix.org/s/LZpzc8jMzbWG9BZ/download?path=/&files=cwp_su_all_44R18.tgz" -O cwp_su_all_44R18.tgz

In the future, will this work with 44R20 etc.? The "LZpzc8jMzbWG9BZ" looks like an encoding that would be likely to change.

**Comment by JohnWStockwellJr** [comment]
That, I do not know. The distribution site is a third party commercial storage site. This is thanks to the University of Pau in France. 

John Stockwell
Visit: https://wiki.Seismic-Unix.org
The new home of Seismic Un*x
